db/migrate/20200430002042_create_work_experiences.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkExperiences < ActiveRecord::Migration[5.2]
+  def change
+    create_table :work_experiences do |t|
+      t.references :student, foreign_key: true, null: false
+      t.references :company, foreign_key: true, null: false
```

Comment:
The companies where the former student worked may not be in the I2T2 database, this should be a string

Reply:
Duplicated